spec/features/customers/customer_can_login_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "When a customer visits the root path" do
+  before do
+    create(:customer, first_name: "Jane", last_name: "Doe", email: "jane@jane.com")
+    visit root_path
+  end
+
+  scenario "the customer should see a log in button or link" do
+    expect(page).to have_selector(:link_or_button, "Log in")
+  end
+
+  scenario "the customer should be redirected to the login path after clicking log in" do
+    click_on "Log in"
+
+    expect(current_path).to eql(login_path)
```

Comment:
expect(page).to_not have_selector(:link_or_button, "Log in")
expect(page).to have_selector(:link_or_button, "Log out")